app-emulation/rkt/rkt-9999.ebuild

```diff
@@ -92,7 +92,7 @@ src_configure() {
 	myeconfargs+=( $(use_enable tpm) )
 
 	myeconfargs+=( --with-stage1-flavors="${STAGE1FLAVORS}" )
-	myeconfargs+=( --with-stage1-default-location="/usr/share/rkt/stage1-${STAGE1FIRST}.aci" )
+	myeconfargs+=( --with-stage1-default-location="$(STAGE1INSTALLDIR)/stage1-${STAGE1FIRST}.aci" )
```

Comment:
I think could should add `--with-stage1-default-images-directory=${STAGE1INSTALLDIR}` to change the lookup path